### ISSUE TITLE
Double search icon

### ISSUE THREAD

**Issue by macolso** [fermyon/developer#208]
In the [Fermyon Cloud developer docs](https://developer.fermyon.com/cloud/index), there appears to be a double search ion. I would expect only one (non-emoji).

![image](https://user-images.githubusercontent.com/48335676/202520381-953d926f-687d-40a1-9cbd-100720407ec9.png)

**Comment by macolso** [comment]
With help from @karthik2804 , realized this was an intermittent repro (wouldn't repro in incognito browser). hard refresh resovled the issue.

**Comment by karthik2804** [comment]
Yes, we should upgrade the static file server but I still think there might be one or two edge cases with assets being cached for the older duration.
